packages/gestalt/src/Contents.js

```diff
@@ -202,25 +202,6 @@ class Contents extends Component<Props, State> {
     }
   };
 
-  calcTopHeight() {
-    if (!window || !document) {
-      return { top: null, height: null };
-    }
-
-    const { scrollBoundaryContainerRef, positionRelativeToAnchor } = this.props;
-
-    // Define the height based the reference to render: ScrollBoundaryContainer or screen viewport
-    const height = scrollBoundaryContainerRef?.offsetHeight ?? window.innerHeight ?? 0;
```

Comment:
Did you try remove only this `scrollBoundaryContainerRef?.offsetHeight` of code?
You are right, when I created this code set the `scrollBoundaryContainerRef?.offsetHeight` as a controller of size.